internal/controller/management/projectconfigs/project_configs.go

```diff
@@ -0,0 +1,311 @@
+package projectconfigs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"time"
+
+	multierr "github.com/hashicorp/go-multierror"
```

Comment:
No need to introduce this. `k8s.io/apimachinery/pkg/util/errors` has an aggregated error (search for `kerrors` in code base to see examples).